src/styles/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$transitioin-smooth: 300ms;
+$card-hover-scale: 1.2;
+$card-accent-color: #34568b;
+$nav-link-accent-color: #00acdc;
```

Comment:
do u actually need that much of variables?
![image](https://user-images.githubusercontent.com/127235037/234596949-01eecf98-2d76-49b5-a61a-51d462f8564f.png)

Reply:
Probably no but i really love variables :D